### ISSUE TITLE
Multiple errors when starting container

### ISSUE THREAD

**Issue by Bazmundi** [SE-Stuttgart/xstampp-4.0#2]
I am running Windoze 11, with Docker Desktop and running Ubuntu on the WSL.

Getting "Error: Invalid or corrupt jarfile app.jar" on all but ext-nginx-1 and postgresql.  

ext-nginx-1 seems to restart at a periodicity slower than the rest, reports:

```
ext-nginx-1       | 2024/07/06 02:40:38 [emerg] 1#1: host not found in upstream "push" in /etc/nginx/conf.d/xstampp.conf:18
ext-nginx-1       | nginx: [emerg] host not found in upstream "push" in /etc/nginx/conf.d/xstampp.conf:18
```

Container postgresql is spitting out:

```
ext-postgresql-1  | chmod: changing permissions of '/var/lib/postgresql/data': Operation not permitted                                                                                                     
ext-postgresql-1  | The files belonging to this database system will be owned by user "postgres".
ext-postgresql-1  | This user must also own the server process.
```

Multiple attempts by contrainer to restart fail.   Multiple restart of the container same problem.

**Comment by wagnerst** [comment]
Dear Bazmundi,

as this was a university project, we unfortunately do not have the resources for active maintenance. I am sorry!

**Comment by Bazmundi** [comment]
> Dear Bazmundi,
> 
> as this was a university project, we unfortunately do not have the resources for active maintenance. I am sorry!

Yep,  understood.  So you might want approach MIT about removing the reference to this tool from their [STPA site](https://psas.scripts.mit.edu/home/stamp-tools/), since they are promoting tools to support safety engineers in production environments?  Assuming you may have promoted your dabbling to them in the first instance?

**Comment by wagnerst** [comment]
Thank you for the hint!


Am 09.07.2024 um 11:01 schrieb Asterion Daedalus ***@***.***>:

﻿

Dear Bazmundi,

as this was a university project, we unfortunately do not have the resources for active maintenance. I am sorry!

Yep, understood. So you might want approach MIT about removing the reference to this tool from their STPA site<https://psas.scripts.mit.edu/home/stamp-tools/>, since they are promoting tools to support safety engineers in production environments?

**Comment by Bazmundi** [comment]
> Dear Bazmundi,
> 
> as this was a university project, we unfortunately do not have the resources for active maintenance. I am sorry!

Yep, so tried on a standard Linux distro with same result.  

I guess that the idea of docker is the means to pull down and run software with all dependencies seems to mean this never ran?  Noting the wget address is also dead?

Probably needs the repositary deleted?

**Comment by Bazmundi** [comment]
Yep, tried building and build also fails.

**Comment by robi-y** [comment]
I could built it a while back, however it was limited to an older Eclipse version #15

**Comment by robi-y** [comment]
I could built it a while back, however it was limited to a previous Eclipse version https://github.com/SE-Stuttgart/XSTAMPP/pull/15

**Comment by Bazmundi** [comment]
Hmm. I downloaded eclipse version cited. I'll keep working on it because
this tool seems to include more of the up to date stpa concepts.
